nmrfx-processor-gui/src/main/resources/fxml/AttributesController.fxml

```diff
@@ -0,0 +1,420 @@
+                                      onAction="#sliceAction" text="Center slices at crosshair"/>
+                            <CheckBox fx:id="useDatasetColorCheckBox" mnemonicParsing="false"
+                                      onAction="#sliceAction" text="Use dataset color for slices" selected="true"/>
+                            <GridPane BorderPane.alignment="CENTER" GridPane.rowIndex="0"
+                                      GridPane.columnIndex="0" GridPane.columnSpan="1">
+                                <children>
+                                    <Label text="Slice 1"
+                                           GridPane.columnIndex="0" GridPane.rowIndex="0"
+                                    />
+                                    <CheckBox fx:id="slice1StateCheckBox" mnemonicParsing="false"
+                                              onAction="#sliceAction" text="" selected="true"
+                                              GridPane.columnIndex="1" GridPane.rowIndex="0"
+                                    />
+                                    <ColorPicker fx:id="slice1ColorPicker" onAction="#sliceAction"
+                                                 promptText="Slice 1 Color"
+                                                 GridPane.columnIndex="2" GridPane.rowIndex="0"
+                                    />
+                                    <Label text="Slice 2"
+                                           GridPane.columnIndex="0" GridPane.rowIndex="1"
+                                    />
+                                    <CheckBox fx:id="slice2StateCheckBox" mnemonicParsing="false"
+                                              onAction="#sliceAction" text="" selected="false"
+                                              GridPane.columnIndex="1" GridPane.rowIndex="1"
+                                    />
+                                    <ColorPicker fx:id="slice2ColorPicker" onAction="#sliceAction"
+                                                 promptText="Slice 2 Color"
+                                                 GridPane.columnIndex="2" GridPane.rowIndex="1"
+                                    />
+                                </children>
+                            </GridPane>
+                            <GridPane BorderPane.alignment="CENTER" GridPane.rowIndex="0"
+                                      GridPane.columnIndex="0" GridPane.columnSpan="1">
+                                <columnConstraints>
+                                    <ColumnConstraints minWidth="50.0" prefWidth="50.0"/>
+                                    <ColumnConstraints minWidth="50.0" prefWidth="50.0"/>
+                                    <ColumnConstraints minWidth="50.0" prefWidth="50.0"/>
+                                </columnConstraints>
+                                <children>
+                                    <Label text="X Oƒfset" GridPane.columnIndex="0" GridPane.rowIndex="0"/>
```

Comment:
for some reason one of the f was fancy 😆 
```suggestion
                                    <Label text="X Offset" GridPane.columnIndex="0" GridPane.rowIndex="0"/>
```

Reply:
That's my cast bumping the Alt key when I type